packages/web/components/cards/generic-main-card.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ReactNode } from "react";
+
+import { IconLink } from "~/components/cards/icon-link";
+
+export const GenericMainCard: React.FC<{
```

Comment:
Again, IMO `FunctionComponent` is more readable and standard (though of course we all know what this is)